tests/unit/test_charm.py

```diff
@@ -243,7 +243,7 @@ def test_given_default_config_when_config_changed_then_upf_route_is_created(
 
         patch_exec.assert_called_with(
             command=["ip", "route", "replace", upf_ip_address, "via", upf_gateway],
-            timeout=30,
+            timeout=300,
```

Comment:
5min to change a route? It seems like if it takes more than a couple of seconds, it's clear that we have a problem and there's no reason to wait longer.

Reply:
The timeout is hardcoded for all commands executed in the container. In this case, this will not change anything because the command returns immediately.

We can refactor the code to permit different timeouts per commands, but I do not see the value. We expect the timeout to never be hit.